CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## **[Unreleased]**
+- [#1602](https://github.com/wasmerio/wasmer/pull/1602) Fix panic when calling host functions with negative numbers in certain situations
```

Comment:
Could we get a test for that?

Reply:
Sure, any recommendations on where a test this basic should go? 😆

Reply:
Hrm, I don't really see a good place for it. My first choice is to create a new place, `tests/api` which tests lib/api or `tests/regression` which contains tiny tests to ensure that we don't have regressions on fixed PRs. I somewhat prefer the latter because I think we'll want a grab-bag directory for just "PRs we've fixed in the past" and tests can have names like `pr1602.rs`.

Reply:
Oh didn't see this comment until now... hmm that's a good idea, I put in the api/tests for now because it is testing the API.

It would be a good idea to have a place to track all the tests we added this way as well as having more explicit regression tests

Reply:
Oh, so the `tests/api` are the unit tests on the api crate so it's at `lib/api/tests`